ibm-messaging-mq-cloud-showcase-app/QMGR-docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+services:
+  mq:
+    image: "icr.io/ibm-messaging/mq:latest"
+    ports:
+      - 9443:9443
+      - 1414:1414
+    environment:
+      - LICENSE=accept
+      - MQ_QMGR_NAME=QM1
+      - MQ_APP_PASSWORD=${APP_PASSWORD}
```

Comment:
use the syntax ${APP_PASSWORD:?err} to signal failure when password has not been provided.